cmd/remove.go

```diff
@@ -36,7 +36,10 @@ use 'hosts disable' instead.
 		profile, _ := cmd.Flags().GetString("profile")
 		dst, _ := cmd.Flags().GetString("host-file")
 
-		err := host.Remove(dst, profile)
+		err := host.RemoveProfile(&host.RemoveProfileOptions{
```

Comment:
I prefer to have positional parameters in this case. I use to add struct options only for optional params or params with useful defaults. But in this case, all are required.

```suggestion
		err := host.RemoveProfile(dst, profile)
```